rosbag2_compression/test/rosbag2_compression/test_sequential_compression_writer.cpp

```diff
@@ -148,8 +147,10 @@ class SequentialCompressionWriterTest : public TestWithParam<uint64_t>
   std::unique_ptr<rosbag2_cpp::Writer> writer_;
 
   std::string serialization_format_;
-  uint64_t fake_storage_size_{};
+  std::atomic<uint32_t> fake_storage_size_{0};  // Need to be atomic for cache update since it
```

Comment:
just missed?

```suggestion
  std::atomic<uint64_t> fake_storage_size_{0};  // Need to be atomic for cache update since it
```

Reply:
It is safe to have `uint32_t` type there because we increase `fake_storage_size_` by **one** with every message written. That is not an actual storage size and is going to be very small numbers during the test.

Reply:
Fixed in the https://github.com/ros2/rosbag2/pull/1643/commits/2cbfc29187a6b308d7968515a2071e8e5ef84fbc commit